File: 0812-rotate-string/0812-rotate-string.py
class Solution(object):
    def rotateString(self, s, goal):
        """
        :type s: str
        :type goal: str
        :rtype: bool
        """
        for i in range(len(s)+1):
            if s[i:]+s[:i]==goal:
                return True
        return False     

File: 0812-rotate-string/0812-rotate-string.java
class Solution {
    public boolean rotateString(String s, String goal) {
        if(s.length()!=goal.length()){
            return false;
        }
        for(int i=0;i<=s.length();i++){
            if((s.substring(i)+s.substring(0,i)).equals(goal)){
                return true;
            }
        }
        return false;
    }
}